lib/fauxpaas/logged_release.rb

```diff
@@ -35,10 +35,10 @@ def initialize(user, time, signature)
     end
 
     def to_s
-      "#{formatted_time}: #{user} #{archive_s(signature.source)} " \
-        "w/ #{archive_s(signature.deploy)}\n" \
-        "  #{archive_s(signature.unshared)}\n" \
-        "  #{archive_s(signature.shared)}"
+      "#{formatted_time}: #{user} #{signature.source.commitish} " \
+        "w/ #{signature.deploy.commitish}\n" \
+        "  #{signature.unshared.commitish}\n" \
+        "  #{signature.shared.commitish}"
```

Comment:
I don't know what commitish is but I see that this allows for the deletion of archive_s.